### DISCUSSION TITLE
Save Plex credentials

### DISCUSSION THREAD

**Discussion by MrOneD** [Taxel/PlexTraktSync#1236] [Q&A] [answered]
Is it possible to save Plex password or set it as param like "plextraktsync -p PLEXPASSWORD" ?

**Comment by simonc56** [comment]
What for ?

**Reply by MrOneD** [reply]
Because always when i want to sync it ask me to enter password, and i want to automate it

**Reply by simonc56** [reply]
This is not normal.
At first run the script is supposed to save your credentials in `.env` file.
What command do you run for sync ?
Can you show the logs of your plex login process ?

**Reply by MrOneD** [reply]
just "plextraktsync" and it always ask me to enter password
here is my .env
![image](https://user-images.githubusercontent.com/98280063/204821974-2feb8c9f-a419-435a-b4bd-06db32351bd6.png)
for login just hit enter but always need to enter password
![image](https://user-images.githubusercontent.com/98280063/204822906-ac4a0c99-a6a7-40bb-839a-fac72a2cae7b.png)
Nothing special in plextraktsync.log

**Reply by simonc56** [reply]
> Nothing special in plextraktsync.log

I don't know. Show us please.

login screenshot is cropped, I can't see the whole process. Your plex token is not written in .env file. It's a problem.

**Reply by MrOneD** [reply]
Thats all from log when i use plextraktsync

```2022-11-29 15:01:15,703 INFO[PlexTraktSync]:PlexTraktSync [0.24.3]
2022-11-29 15:01:40,104 INFO[PlexTraktSync]:Sync Movie sections: [<PlexLibrarySection:movie:Filmy>]
2022-11-29 15:01:40,105 INFO[PlexTraktSync]:Sync Show sections: [<PlexLibrarySection:show:Seriale>]
2022-11-29 15:01:41,293 INFO[PlexTraktSync]:Filmy processed in 0.6 seconds
2022-11-29 15:01:45,207 INFO[PlexTraktSync]:Seriale processed in 3.9 seconds
2022-11-29 15:01:46,169 INFO[PlexTraktSync]:Updated watchlist and/or liked list in 
2022-11-29 15:01:46,172 INFO[PlexTraktSync]:Completed full sync in 6.1 seconds

**Comment by MrOneD** [comment, marked as answer]
OK i resolve problem:

1) I get my X-Plex-Token - https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/
2) Add it manually to .env file

Unfortunately, I still dont know why it didnt complete itself.

**Reply by glensc** [reply]
> Unfortunately, I still dont know why it didnt complete itself.

because you didn't report bug as issue, but instead opened discussion.
